docs/intro_tutorial.rst

```diff
@@ -135,18 +122,11 @@ information. For instance, the GHZ state
 
 is a well-known 3-qubit quantum state. We can invoke this using :code:`|toqito⟩` as
 
-.. code-block:: python
+.. jupyter-execute::
+
+     from toqito.states import ghz
+     ghz(2, 3)
 
-    >>> from toqito.states import ghz
-    >>> ghz(2, 3)
-    array([[0.70710678],
-           [0.        ],
-           [0.        ],
-           [0.        ],
-           [0.        ],
-           [0.        ],
-           [0.        ],
-           [0.70710678]])
```

Comment:
@RajdeepAher You forgot the code block below this. 

This was from a different PR that showed up after you rebased your branch.